## 02_fix_ld-flags.diff by Ryan Kavanagh <ryanakca@kubuntu.org>
## Description: We don't need useless linking
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -30,7 +30,7 @@ knmap_SOURCES = main.cpp knmap.cpp mainw
 					profiledialog.cpp loggingoptions.cpp simpleoptions.cpp pandsoptions.cpp whatsthis.cpp \
 					tabwidgetptrlist.cpp tabwidgetdata.cpp scanwidget.cpp indexwidget.cpp scannamedialog.cpp \
 			scanstack.cpp scanmonitor.cpp scanmonitorevent.cpp nmapoutputbuffer.cpp
-knmap_LDFLAGS = $(KDE_RPATH) $(all_libraries)
+knmap_LDFLAGS = #$(KDE_RPATH) $(all_libraries)
 knmap_LDADD = $(LIB_KIO) $(LIB_KHTML)
 
 # this is where the desktop file will go 
